### DISCUSSION TITLE
Terraformer Kubernetes finds deployments but gets connection refused

### DISCUSSION THREAD

**Discussion by siegelpeter** [GoogleCloudPlatform/terraformer#1402] [General]
Hi

I trying to use terraformer to import all my kubernetes deployments.
It seems to be able to find my deployments allright but it cannot access them.
2022/08/02 22:54:23 Get "http://localhost/apis/apps/v1/namespaces/ingress-nginx/deployments/ingress-nginx-nginx-ingress-controller": dial tcp 127.0.0.1:80: connect: connection 
refused

as this is an aks cluster it should try the server hostname from the kubeconfig.

This is my provider config.
```
terraform {
  required_providers {
    kubernetes = {
      source  = "hashicorp/kubernetes"
      version = ">= 2.9.0"
    }
    helm = {
      source = "hashicorp/helm"
    }
  }
}
provider "kubernetes" {
  config_path = "/home/xxx/.kube/config"
  host                   = data.aws_eks_cluster.cluster.endpoint
}
provider "helm" {
  kubernetes {
    config_path = "/home/xxx/.kube/config"
  }
}

```
Can somebody point me in the right direction?

**Comment by jan4984** [comment]
I did lot google search, finally find the answer. We need KUBE_CONFIG_PATH env, the config_path in .tf file not work.
